cm/cm_hdd_cdd/worker.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# from pprint import pprint
+from BaseCM import cm_base as cm_base
+
+from hddcdd import hdd_cdd_stats
+
+# from BaseCM import cm_input as cm_input
+
+
+app = cm_base.get_default_app("hddcdd")
+schema_path = cm_base.get_default_schema_path()
+
+
+@app.task(base=cm_base.CMBase, bind=True, schema_path=schema_path)
+def cm_hddcdd(self, selection: dict, rasters: list, params: dict):
+    """This is a calculation module that multiplies the raster by an factor.
+    If there is no raster, we raise a value error.
+    If there are many rasters, we select the first one.
+    """
+    # print("=" * 50)
+    # print("CM-HDD-CDD")
+    # print("Selection:")
+    # pprint(selection)
+    # print("Rasters:")
+    # pprint(rasters)
+    # print("Params:")
+    # pprint(params)
+
+    # self.validate_params(params)
```

Comment:
I comment them out otherwise the log is quite verbose for each client request, but I keep it as comment if someone need to locally test what the function is receiving from the server.

Reply:
ok, let's keep these lines as comments but I think it would be a good idea to uncomment this line `self.validate_params(params)`